chromaui/chromatic-cli
# Add new invalid sb base dir error message and validator function

We will now check if the `storybookBaseDir` is valid before letting the build continue. If it is not defined but needs to be, or is defined but does not point to the correct location, the build will fail and an error will be displayed.

## QA
1. Use an app that has TurboSnap enabled
2. Commit a visual change
3. Using the canary build, run a build with `--only-changed` and an invalid `--storybook-base-dir`
   - e.g. `npx chromatic@<CANARY_VERSION> --only-changed --storybook-base-dir "src/foobar"`
4. Verify that the build fails with an error message indicating that the Storybook base dir is invalid
   - Prior to this change, the build would complete successfully, incorrectly detecting no visual changes


<details>
  <summary>📦 Published PR as canary version: <code>11.0.0--canary.921.8007040697.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install chromatic@11.0.0--canary.921.8007040697.0
  # or 
  yarn add chromatic@11.0.0--canary.921.8007040697.0
  ```
</details>
